app/Main.hs

```diff
@@ -1,3 +1,5 @@
+{-# LANGUAGE RecordWildCards #-}
```

Comment:
You don't need to use this language extension because it's enable for whole package here: https://github.com/serokell/importify/blob/e8878e721bc91ff3f5096f327c81375a768fc62e/importify.cabal#L45
There's set of extensions we agree to use by default. `-XRecordWildCards` is one them.